spec/controllers/assignments_controller_spec.rb

```diff
@@ -204,6 +204,41 @@
     end
   end
 
+  describe "display student username", :vcr, type: :view do
+    before do
+      organization.users.push(create(:user, uid: 90, token: "asdfsad4333"))
+      organization.roster = create(:roster)
+      organization.roster.roster_entries.push(RosterEntry.create(
+                                                identifier: "student",
+                                                roster: organization.roster,
+                                                user_id: organization.users.last
+      ))
```

Comment:
Nit-pick: please use indented style as described in https://github.com/education/classroom/pull/1473

For example:
```diff
-      organization.roster.roster_entries.push(RosterEntry.create(
-	                                                identifier: "student",
-	                                                roster: organization.roster,
-	                                                user_id: organization.users.last
-	      ))
+      organization.roster.roster_entries
+        .push(RosterEntry.create(
+          identifier: "student",
+          roster: organization.roster,
+          user_id: organization.users.last
+        ))
```

Also it would be cool if you could make your editor use 2 spaces in place of tabs 😅

Reply:
I'll make a new PR for that.